l10n_ch_qr_bill_scan/wizard/account_invoice_import.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2020 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.modules.module import get_resource_path
+
+from ..tools import QR
+
+logger = logging.getLogger(__name__)
+
+try:
+    import cv2
+    import numpy as np
+    import pdf2image
+    from pyzbar.pyzbar import ZBarSymbol, decode
+except ImportError:
+    # Necessary libraries to decode QR from pdf are
+    # not installed
+    logger.warning(
+        "Swiss QR detection in Vendor Bill pdf disabled."
+        " To enable it, please install pyzbar, pdf2image, cv2 and numpy."
+    )
+    decode = False
+
+
+class AccountInvoiceImport(models.TransientModel):
+    _inherit = "account.invoice.import"
+
+    invoice_scan = fields.Text(string="Scan of the invoice")
+    invoice_file = fields.Binary(string="PDF, PNG or XML Invoice", required=False)
```

Comment:
```suggestion
    invoice_file = fields.Binary(required=False)
```
if we override args we keeping only those we are overriding

Reply:
agree with you! forgot to check original field definition in parent model

Reply:
but probably it's also overriding of field's string (we add PNG support)
https://github.com/yankinmax/edi/blob/1ff1ee4c84b3165111de712d97ec85c6a2b142cd/account_invoice_import/wizard/account_invoice_import.py#L23

Reply:
ok